.github/workflows/update-maintainers.yml

```
@@ -0,0 +1,245 @@
+          else
+            echo "No PR is currently open"
+          fi
+
+      - name: Fetch current PR's branch
+        if: steps.open_pr_info.outputs.number
+        run: |
+          git fetch origin "$pr_branch"
+          git branch --set-upstream-to "origin/$pr_branch"
+
+      - name: Create Pull Request
+        id: create-pr
+        if: steps.check-changes.outputs.has_changes == 'true' || github.event.inputs.create_pr == 'true'
+        env:
+          GH_TOKEN: ${{ steps.app-token.outputs.token || github.token }}
+          title: "maintainers: update generated/all-maintainers.nix"
+          commit_body: |
+            Automated update of the master maintainers list combining:
+            - Nixvim specific maintainers from lib/maintainers.nix
+            - Nixpkgs maintainers referenced in Nixvim modules
+
+            Changes: ${{ steps.check-changes.outputs.changes_summary || 'No content changes' }}
+
+            Generated by: flake/dev/generate-all-maintainers/generate-all-maintainers.py
+          pr_url: ${{ steps.open_pr_info.outputs.url }}
+          pr_num: ${{ steps.open_pr_info.outputs.number }}
+          pr_body: |
+            ## 📋 Summary
+
+            This PR updates the master maintainers list (`generated/all-maintainers.nix`) which combines:
+            - **Nixvim specific maintainers** from `lib/maintainers.nix`
+            - **Nixpkgs maintainers** referenced in Nixvim modules
+
+            ## 🔄 Changes
+
+            **Statistics:** ${{ steps.check-changes.outputs.changes_summary || 'No content changes (format/comment updates only)' }}
+
+            The updated list includes all maintainers needed for review assignments across the Nixvim project.
+
+            ## 🤖 Automation
+
+            - **Generated by:** `flake/dev/generate-all-maintainers/generate-all-maintainers.py`
+            - **Trigger:** ${{ github.event_name == 'schedule' && 'Scheduled weekly update' || 'Manual workflow dispatch' }}
+            - **Validation:** File syntax verified with `nix-instantiate --eval`
+
+            ## 📚 Usage
+
+            This file can be imported and used for maintainer lookups:
+            ```nix
+            let allMaintainers = import ./generated/all-maintainers.nix; in
+            # Access any maintainer by name: allMaintainers.username
+            ```
+
+            ---
+            🤖 *This PR was automatically created by the [update-maintainers workflow](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})*
+        run: |
+          git add generated/all-maintainers.nix
+          git commit -m "$title" -m "$commit_body"
+
+          echo "Pushing to remote branch $pr_branch"
+          git push --force --set-upstream origin "$pr_branch"
+
+          if [ -z "$pr_num" ]; then
+            echo "Creating new pull request."
+            PR_URL=$(gh pr create \
+              --title "$title" \
+              --body "$pr_body")
```

Personally, I find subshells, command substitution, etc clearer like this:


```suggestion
            PR_URL=$(
              gh pr create \
                --title "$title" \
                --body "$pr_body"
            )
```